src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -503,6 +512,17 @@ def open_pr_comment_workflow(pr_id: int) -> None:
 
         function_names = language_parser.extract_functions_from_patch(file.patch)
 
+        if file_extension in ["js", "jsx"]:
```

Comment:
this is a bit nit-picky... but... thoughts on moving `extra` to a dictionary, then invoking logger.info once? (not sure how temporary these logs are)

```py
log_name = "github.open_pr_comment"
extra = { ... }

# ... remains the same ...

if file_extension in ["js", "jsx"]:
  extra["has_function_names"] = bool(function_names)
  log_name += ".javascript"
else: 
  log_name += ".file_extension"

logger.info(log_name, extra);
```

this way will also mean only 1 log per JS file instead of 2.

Reply:
these are very temporary, but sure!